README.md

```diff
@@ -15,7 +15,26 @@ pip install pytest-inmanta-lsm
 This plugin is used to push code to a remote orchestrator and interact with it, via the LSM north-bound-api
 It requires an LSM enabled orchestrator, with no ssl or authentication enabled, in a default setup and ssh access to the orchestrator machine, with a user that has sudo permissions.
 
-## Usage
+## Remote access configuration
+
+To setup the orchestrator and run your test with it, pytest-inmanta-lsm needs to have access to both the api, and the file system of the orchestrator.  There exists a number of options to configure pytest-inmanta-lsm to reach these.  All of them are shortly presented in the [Options and environment variables section](#options-and-environment-variables), but let's go through a few concrete examples here below:
+
+1. The orchestrator has been installed with rpm, and has been started with systemd.  You have ssh access to the host where it is running.  You need to tell pytest-inmanta-lsm the following things:
+    - Where is the api? Use `--lsm-host=<srv-ip>` and `--lsm-srv-port=<srv-port>` (or `INMANTA_LSM_HOST=<srv-ip>` and `INMANTA_LSM_SRV_PORT=<srv-port>`) options.  Where `<srv-ip>` should be replaced with the ip of the host where the orchestrator is running and `<srv-port>` the port on that host where the orchestrator is listening.
+    - How to open a shell on the host? Use `--lsm-rsh=ssh` and `--lsm-rh=<ssh-user>@<ssh-ip>` (or `INMANTA_LSM_REMOTE_SHELL=ssh` and `INMANTA_LSM_REMOTE_HOST=<ssh-user>@<ssh-ip>`) options, telling pytest-inmanta-lsm to rely on ssh to access the remote host at `<ssh-ip>` and to login using user `<ssh-user>`.  Additional ssh options, like a different port, may be passed in `--lsm-rsh` option.  The ssh user must either be `inmanta` or be part of the `sudo` group to be able to become `inmanta`.  In this scenario, `<srv-ip>` and `<ssh-ip>` are identical.
```

Comment:
I wonder if it would help to add a code block with the consolidated config, to support copy paste and skip reading

```bash
# For RPM installed orchestrator
pytest tests --lsm-host=<srv-ip> --lsm-rh=<ssh-user>@<ssh-ip> --lsm-srv-port=8888 
```